core/src/main/java/io/github/fourlastor/game/coordinates/Hex.java

```diff
@@ -65,4 +64,67 @@ public void set(Hex other) {
     public boolean sameAxisAs(Hex hex) {
         return hex.cube.x == cube.x || hex.cube.y == cube.y || hex.cube.z == cube.z;
     }
+
+    public Hex offset(Direction direction, int amount) {
+        return new Hex(cube.cpy().add(direction.cube.cpy().scl(amount)));
+    }
+
+    public enum Direction {
+        S(0, -1, 1),
+        SE(1, -1, 0),
+        NE(1, 0, -1),
+        N(0, +1, -1),
+        NW(-1, 1, 0),
+        SW(-1, 0, 1),
+        NONE(0, 0, 0),
```

Comment:
I think `NONE` shouldn't exist (unless we have a use case for it other than as a default)

Reply:
I do recall there being some usecase for it in PokeWilds but I don't remember what it was.